RecDP/pyrecdp/primitives/llmutils/pii_remove.py

```diff
@@ -130,54 +79,15 @@ def pii_remove(args):
             logging.StreamHandler()
         ]
     )
-    logger.info(f"** The job is running with the following arguments: **\n{args}\n **** ")
-
-    logger.info(f" ===== Loading {args.path} =====")
-    data_files = args.data_files if args.data_files else None
-    ds = load_dataset(args.path,  data_files=data_files, split=args.split)
-    if args.text_column != "content":
-        ds = ds.rename_column(args.text_column, "content")
-
-    logger.info(f" ===== Applying PII detection =====")
-    ds_pii = ds.map(
-        scan_pii_batch, batched=True, batch_size=args.batch_size, num_proc=args.num_proc, load_from_cache_file=False
-    )
-    logger.info(f"Dataset info after PII detection:\n{ds_pii}")
-    logger.info(f"Number of samples that contained PII: {sum(ds_pii['has_secrets'])}")
-    logger.info(f"Total number of secrets found: {sum(ds_pii['number_secrets'])}")
 
-    # redact PII in the dataset
-    logger.info(f" ===== Applying PII redaction =====")
-    random.seed(args.seed)
-
-    # we use random replacements by default
-    replacements = random_replacements()
-    with open("random_replacements.json", "w") as f:
-        json.dump(replacements, f)
-    logging.info(f"Using the following replacements:\n{pformat(replacements)}")
-    ds_pii = ds_pii.map(
-        partial(redact_pii_batch, replacements=replacements),
-        batched=True,
-        batch_size=args.batch_size,
-        num_proc=args.num_proc,
-        load_from_cache_file=False
-    )
-    logging.info(f"Dataset info after PII redaction:\n{ds_pii}")
-
-    logger.info("Removing columns that are not needed for the final dataset")
-    columns = ["content", "modified", "secrets", "has_secrets", "number_secrets"]
-    ds_pii = ds_pii.remove_columns(columns)
-    ds_pii = ds_pii.rename_column("new_content", "content")
-    logger.info(f"Dataset info after removing columns:\n{ds_pii}")
+    args = getArgs()
+    logger.info(f"** The job is running with the following arguments: **\n{args}\n **** ")
 
-    # save the final dataset
+    spark = SparkSession.builder.master("local[{}]".format(args.cpu_per_worker)).appName(
+        "PII Remove").getOrCreate()
```

Comment:
@chaojun-zhang , let's using recdp embedded way to start spark, instead using SparkSession here

Reply:
Is there any reason to use this way?